README.md

```diff
@@ -1,70 +1,202 @@
+<!-- PROJECT DESCRIPTION -->
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+# 📖 [React_Capstone] <a name="about-project"></a>
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+**[React_Capstone]**  is to show famous games and personal favorites, in which you can search for any and see if they are in the hall of fame ༼ つ ◕_◕ ༽つ,
+ and to show the skills that has been learned through out the past five weeks.
 
-### `npm test`
+## 🛠 Built With <a name="built-with"></a>
+- CSS, JavaScript
+- React/Redux
+- Linters
+### Tech Stack <a name="tech-stack"></a>
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+> Describe the tech stack and include only the relevant sections that apply to your project.
 
-### `npm run build`
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+<details>
+<summary>API</summary>
+  <ul>
+    <li><a href="https://rapidapi.com/accujazz/api/rawg-video-games-database">RAWG API</a></li>
+  </ul>
+</details>
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+<!-- Features -->
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+### Key Features <a name="key-features"></a>
 
-### `npm run eject`
+> Describe between 1-3 key features of the application.
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+- **[Filter]** in which the user can search for desired games.
+- **[Show_Games]** in which the user can see famous games in the home page.
+- **[Details]** on click the user can see the detials of a certain game
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+<!-- LIVE DEMO -->
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+## 🚀 Live Demo <a name="live-demo"></a>
```

Comment:
- [ ] please kindly ensure your project is deployed using Heroku, Netlify or GitHub Pages and a link added to readme. please check if it is working properly.
![Screenshot (627)](https://user-images.githubusercontent.com/65298048/203867565-cd26bc54-6b8c-4259-a48d-b6f0ff924302.png)